proposals/mdawson-cpc-charter/CPC-Charter.md

```diff
@@ -242,15 +211,6 @@ the same person. The CPC shall hold annual elections to select a CPC
 Chairperson and Director; there are no limits on the number of terms a
 CPC Chairperson or Director may serve.
```

Comment:
The TC39 has recently implemented a Chair Group, which we are hoping will be quite effective. Based on the scale of this role in the larger foundation we may want to consider this... it would also help distribute the responsibility across projects... also helping with growth of more junior individuals who may not feel comfortable taking on the entire role.

Reply:
Do you have a reference/link with more info on that approach?